hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -109,12 +111,29 @@ def create(project_root: Path,
         else:
             logging.info("Running inside AzureML.")
             logging.info("All results will be written to a subfolder of the project root folder.")
-            run_folder = project_root
-            outputs_folder = project_root / DEFAULT_AML_UPLOAD_DIR
-            logs_folder = project_root / DEFAULT_LOGS_DIR_NAME
+            amulet_keys_not_set = get_amulet_keys_not_set()
+            if len(amulet_keys_not_set) > 0:
+                logging.debug("Assumed not to be an Amulet job since the following environment variables are not set:"
```

Comment:
You have defined a function for that already, `is_amulet_job`?

Reply:
Yes, previously I didn't want to read from the environment variables twice but it's been necessary to change this now anyway since's an additional condition for being an 'amulet job' (which is that either snapshot dir or azure batchai working dir must be set)